SKILL.tsv

```diff
@@ -5201,9 +5201,9 @@ SKILL_20150918_005200	$Increases the damage dealt on an enemy with [High Kick] b
 SKILL_20150918_005201	$Earth Wave: Remove Knockdown
 SKILL_20150918_005202	$[Earth Wave] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
 SKILL_20150918_005203	$Steed Charge: Remove Knockdown
-SKILL_20150918_005204	$[Steed Charge] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
+SKILL_20150918_005204	$[Steed Charge] no longer knocks enemies down. (Only applies if the attribute is set to ON)
 SKILL_20150918_005205	$Doom Spike
-SKILL_20150918_005206	$[Doom Spike] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
+SKILL_20150918_005206	$[Doom Spike] no longer knocks enemies down. (Only applies if the attribute is set to ON)
```

Comment:
For skill descriptions, we are using the convention that it is the skill performing the action/effect, rather than the character because it is done that way in many other games.



Attack an enemy from a short distance. The target gets knocked back when it is struck a certain number of times.